copy_syslibs.py

```
@@ -0,0 +1,42 @@
+print(
+    "Hello! This script copies system libraries to the source, so you can test the new installer "
+    "before we build the binaries ourselves. This will get deleted after we build the binaries."
+)
+
+print(
+    "We need root permissions to run this script. For that reason, we will run 'sudo true' right now and sudo will remember it."
+)
+
+os.system("sudo true")
+
+os.system("mkdir -p rootfs/filesystem/lib")
```

Ruff (0.15.1)

[error] Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)